app/src/components/market/outcome_table/index.tsx

```diff
@@ -64,10 +67,28 @@ export const OutcomeTable = (props: Props) => {
     displayRadioSelection = true,
     outcomeHandleChange,
     outcomeSelected,
+    payouts = [],
     probabilities,
     withWinningOutcome = false,
   } = props
 
+  const WinningOutcome = (props: any) => {
+    const { balance, index } = props
+
+    if (payouts) {
+      const shares = new Big(balance.shares.toString())
+      const redeemable = new BigNumber(shares.mul(payouts[index]).toString())
+      return (
+        <>
+          <div>{payouts[index] < 1 ? `${payouts[index] * 100}% ` : ''}Winning Outcome</div>
+          {`Redeem ${formatBigNumber(redeemable, collateral.decimals)}`}
+        </>
+      )
+    } else {
+      return <></>
+    }
+  }
+
```

Comment:
Message
Winning outcome, or in case of multiple winning outcomes, X% winning outcome, Y% winning outcome ...